packages/docs/stories/GoalsAndGuidelines.mdx

```diff
@@ -20,9 +20,9 @@ We stay ahead, making our design system more flexible for our brand's future nee
 These principles help us to achieve our goals.
```

Comment:
Can you use the order like in the presentation: Progressive, Unified, Connected
Or use the file from #internal https://github.com/synergy-design-system/Internal/blob/main/docs/guiding_principles.md